benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -346,24 +381,31 @@ int main(int argc, char *argv[])
 
 #ifdef KOKKOS_ENABLE_SERIAL
   using Serial = Kokkos::Serial::device_type;
-  REGISTER_BENCHMARK(ArborX::BVH<Serial>);
+  REGISTER_BENCHMARK(ArborX::BVH<Serial>, 1e3, 1e4, 1e5);
```

Comment:
No, no, no. What's the point of this? Why hardcode the values here, and not inside `REGISTER_BENCHMARK`?

Reply:
We (might and currently do) want to use different values for different backends.

Reply:
Ah, did not notice that at all, this was not in the original patch. I guess you changed it. Hmm. This really spreads the changes.